deps/rabbit/src/rabbit.erl

```diff
@@ -1123,7 +1123,7 @@ log_broker_started(Plugins) ->
 
 print_banner() ->
     Product = product_name(),
-    Version = product_version(),
+    Version = [product_version(), ", ", rabbit_misc:platform_and_version()],
```

Comment:
This line is specific to the product version, and adding OTP version here does not make sense.

Maybe that's why we have the Erlang version printed a few lines above, e.g.:

```
 Starting RabbitMQ 3.8.10+8.g5247909.25+stream on Erlang 23.2.3
 Copyright (c) 2007-2021 VMware, Inc. or its affiliates.
 Licensed under the MPL 2.0. Website: https://rabbitmq.com

  ##  ##      RabbitMQ 3.8.10+8.g5247909.25+stream
  ##  ##
  ##########  Copyright (c) 2007-2021 VMware, Inc. or its affiliates.
  ######  ##
  ##########  Licensed under the MPL 2.0. Website: https://rabbitmq.com
```

Is the Erlang version listed a few lines above not enough? Do users start copying from the rabbit ASCII art down?

The problem with this change is that we have the Copyright notice 2 lines below, so adding the Erlang version here, then the VMware copyright doesn't seem right. By that logic, listing the Erlang version a few lines above, then the copyright, isn't right either.

Why don't we list the Erlang version, OS & the OpenSSL versions below the rabbit ASCII, between `Licensed ...` & `Doc guides...`, like so:

```
 Starting RabbitMQ 3.8.10+8.g5247909.25+stream on Erlang 23.2.3
 Copyright (c) 2007-2021 VMware, Inc. or its affiliates.
 Licensed under the MPL 2.0. Website: https://rabbitmq.com

  ##  ##      RabbitMQ 3.8.10+8.g5247909.25+stream
  ##  ##
  ##########  Copyright (c) 2007-2021 VMware, Inc. or its affiliates.
  ######  ##
  ##########  Licensed under the MPL 2.0. Website: https://rabbitmq.com

  Erlang 23.2.3
  Ubuntu 20.04.1 x86_64
  OpenSSL 1.1.1f

  Doc guides: https://rabbitmq.com/documentation.html
  Support:    https://rabbitmq.com/contact.html
  Tutorials:  https://rabbitmq.com/getstarted.html
  Monitoring: https://rabbitmq.com/monitoring.html
```

Reply:
We can do that. We should be defensive if OpenSSL version cannot be inferred, e.g. Erlang was built without OpenSSL support.

Reply:
That sounds like a reasonable compromise.

I would even be OK to not include the platform architecture if it's difficult to get it. I think we get some (if not all) of the OS details in `rabbitmqctl status`.

Part of this, I would like to see us to drop the `on Erlang 23.2.3` part from the `Starting` line. We can rename the PR to **Add useful environment details to the Startup banner**.